datashuttle/utils/utils.py

```diff
@@ -31,7 +54,7 @@ def raise_error(message: str) -> None:
     raise BaseException(message)
 
 
-def get_appdir_path(project_name: str) -> Path:
+def get_appdir_path(project_name: str) -> Tuple[Path, Path]:
     """
     It is not possible to write to program files in windows
```

Comment:
I thought all logs were saved in the project folder? I think we want this (or in `~/.datashuttle`), not in Program Files.